config/rbac/role.yaml

```diff
@@ -83,32 +98,6 @@ rules:
   - get
   - patch
   - update
-- apiGroups:
-  - workload.nephio.org
-  resources:
-  - amfdeployments
```

Comment:
Oh, this is probably wrong too. Resource names should be all lowercase. Resource names are different than Kind names - they are related but different. Resources relate to the REST calls and get mapped to Kind types. You can theoretically have multiple resources utilizing the same Kinds (though in practice I don't think we do that anywhere in K8s usually).